### DISCUSSION TITLE
possibility to do fan speed tower test

### DISCUSSION THREAD

**Discussion by Mario-vminion** [OrcaSlicer/OrcaSlicer#2906] [Dev]
is it possible to do a fan speed tower test to look what's the best fan speed for overhang ...
like this one (https://www.printables.com/model/200347-ultimate-fan-speed-test-v3)

**Comment by MarioL77** [comment]
I join the request, while the tower can be any model, an existing temperature tower can be used with only the ability to calibrate the ventilation rate in 5% increments and 10% increments?
